app/views/unattended/provisioning_templates/finish/windows_default_finish.erb

```diff
@@ -44,89 +45,104 @@ description: |
 <% if @host.pxe_build? %>
   set ctr=0
   set nettimeout=10
+<% end -%>
 
-  (echo Updating time)
-  (sc config w32time start= auto)
-  sc start w32time
-  ::ipconfig /renew
+  <%= snippet 'windows_network' %>
 
   <% if host_param('ntpServer') %>
-    echo setting time server
+    echo Setting time server
     w32tm /config /manualpeerlist:<%= host_param('ntpServer') %> /syncfromflags:manual /update
   <% end %>
 
-  echo sync time
+  echo Syncing time
   w32tm /resync
   w32tm /resync
 
   <% if host_param('computerDomain') -%>
     <% if host_param('domainAdminAccount').present? && host_param('domainAdminAccountPasswd').present? -%>
-      echo performing secure domain join
+      echo Performing secure domain join
       powershell.exe -OutputFormat text -command Add-Computer -DomainName '<%= host_param('computerDomain') -%>' -Credential (New-Object -TypeName System.Management.Automation.PSCredential '<%= host_param('domainAdminAccount') -%>', (ConvertTo-SecureString -String '<%= host_param('domainAdminAccountPasswd') -%>' -AsPlainText -Force)) <% if host_param('computerOU').present? -%>-OUPath '<%= host_param('computerOU') -%>'<% end -%>
     <% else %>
       <% if host_param('machinePassword').present? %>
-        echo performing unsecure domain join
+        echo Performing unsecure domain join
         powershell.exe -OutputFormat text -command Add-Computer -Domain '<%= host_param('computerDomain') -%>' -Options UnsecuredJoin,PasswordPass -Credential (New-Object -TypeName System.Management.Automation.PSCredential $null, (ConvertTo-SecureString -String '<%= host_param('machinePassword') -%>' -AsPlainText -Force))
       <% end %>
     <% end %>
   <% end %>
 
+  <%#
+  Adjust power setting to 'High performance'
+  -%>
+  powershell /c "powercfg /setactive 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c"
```

Comment:
Is this really appropriate for this PR? I can't speak for Windows admins since I rarely touch it, but doing this unconditionally may not be correct for users.